src/app/container/container.component.ts

```diff
@@ -146,10 +146,27 @@ export class ContainerComponent extends Entry implements AfterViewInit {
     // messy prototype for a carousel https://developers.google.com/search/docs/guides/mark-up-listings
     // will need to be aggregated with a summary page
     this.schema = {
-      '@type': 'ListItem',
-      'position': this.tool.id,
-      'url': this.shareURL
+      '@type': 'SoftwareApplication',
+      'description': this.tool.description,
+      'name': this.tool.name,
+      'softwareVersion': this.defaultVersion,
+      'url': window.location,
+
+      'audience': 'Bioinformaticians',
+      'dateModified': this.tool.lastUpdated,
```

Comment:
How does this get rendered? Note that in schema.org the date needs to be specific format.